libmamba/src/specs/version.cpp

```diff
@@ -198,6 +284,124 @@ fmt::formatter<mamba::specs::VersionPartAtom>::format(
     return fmt::format_to(ctx.out(), "{}{}", atom.numeral(), atom.literal());
 }
 
+namespace mamba::specs
+{
+
+    /***********************************
+     *  Implementation of VersionPart  *
+     ***********************************/
+
+    VersionPart::VersionPart()
+        : atoms()
+        , implicit_leading_zero(false)
+    {
+    }
+
+    VersionPart::VersionPart(std::initializer_list<VersionPartAtom> init)
+        : atoms(init)
+    {
+    }
+
+    VersionPart::VersionPart(std::vector<VersionPartAtom> p_atoms, bool p_implicit_leading_zero)
+        : atoms(std::move(p_atoms))
+        , implicit_leading_zero(p_implicit_leading_zero)
+    {
+    }
+
+    auto VersionPart::str() const -> std::string
+    {
+        return fmt::format("{}", *this);
```

Comment:
Is this really what's expected as default string value for a version? Why not `"{}.{}.{}"` with the appropriate values?

Reply:
All the logic is implemented inside the fmt formatter, so this is really just delegating.

Reply:
Ah yes, makes sense.